.github/workflows/tests.yml

```
@@ -1,24 +1,36 @@
-          node-version: 16
-      - name: Install dependencies
-        run: npm ci
-      - name: Run Linter
-        run: npm run lint
-        continue-on-error: true
-      - name: Build packages
-        run: npm run build
+    test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)